HTML/style.css

```diff
@@ -0,0 +1,56 @@
+.level1 {
+    padding: 10px;
+    background-color: #f0f0f0;
+}
+
+.level2 {
+    padding: 10px;
+    background-color: d0d0d0;
+}
+
+.level3 {
+    padding: 10px;
+    background-color: #b0b0b0;
+}
+
+.level4 {
+    padding: 10px;
+    background-color: #909090;
+}
+
+.level5 {
+    padding: 10px;
+    background-color: #707070;
+    color: white;
+}
+
+.first {
+    margin-left: 18%;
+    margin-right: auto;
+    background-color: #fff;
+    font-family: sans-serif;
+    font-size: large;
+  }
+  .first h1 {
+    font-size: 4em;
+  }
+  .first h1 .mundo {
+    text-decoration: line-through;
+  }
+  .first input {
+    background-color: #00F3BB;
+    border: F3BB;
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/17#issuecomment-2189184790>#3949a2</a><span> - 06/25/2024, 03:02 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The border value for the '.first input' class is incorrectly specified as 'F3BB', which is not a valid CSS value.
</div>
<div id="fix">
<b>Fix:</b> Correct the border value to use a valid CSS value, such as '1px solid #F3BB'.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -42,7 +42,7 @@
  .first input {
      background-color: #00F3BB;
 -    border: F3BB;
 +    border: 1px solid #F3BB;
      height: 2.5emmm;
      margin-top: 3;
      padding: 1%;
  }
```
</div>
</details>
</div>
<br>


</div>